app/utils/validation.ts

```diff
@@ -53,6 +53,16 @@ export const timeIsAfter = (otherField, message) => (value, context) => {
   return [true];
 };
 
+export const isValidAllergy =
+  (
+    message = 'La feltet stå tomt hvis du ikke har noen allergier/preferanser'
+  ) =>
+  (value: string) => {
+    const notValidAnswers = ['ingen', 'ingenting', 'nei', 'nope', 'nada'];
+
+    return [!notValidAnswers.includes(value?.toLowerCase()), message];
```

Comment:
if `value` is always a `string`, why do you need the `?`?

Reply:
hmm, well I think it's always a `string`, but I wasn't sure if it somehow could be `undefined`. I believe I got some error, and I therefore added it. But I think it can be removed ..